src/transformers/trainer_tf.py

```diff
@@ -638,7 +638,9 @@ def apply_gradients(self, features, labels, nb_instances_in_global_batch):
                 reduced_features = {
                     k: ft[: self.args.train_batch_size // self.args.n_replicas] for k, ft in features.items()
                 }
-                reduced_labels = labels[: self.args.train_batch_size // self.args.n_replicas]
+                reduced_labels = {
+                    k: ft[: self.args.train_batch_size // self.args.n_replicas] for k, ft in labels.items()
```

Comment:
```suggestion
                    k: lbl[: self.args.train_batch_size // self.args.n_replicas] for k, lbl in labels.items()
```
ft was an abbreviation of feature in the code you pasted, it makes less sense here.